### PR TITLE
Local setup docs

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Local setup docs

## Type of change

- Documentation update

## Testing steps

N/A

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

**Review by juandspy — APPROVED** [review]

File: docs/local_setup_bare_metal.md [hunk 1]
```diff
@@ -11,4 +11,553 @@ nav_order: 14
...
+Type "help" for help.
+
+
+
+postgres=# create database aggregator;
+CREATE DATABASE
+postgres=# \q
+```
+
+Check if migration have not happed (yet):
```
Comment (Bee-lee) [inline docs/local_setup_bare_metal.md:392]: happened

File: docs/local_setup_bare_metal.md [hunk 2]
```diff
@@ -11,4 +11,553 @@ nav_order: 14
...
+
+```
+$ ./bootstrap.sh
+```
+
+Please note that sometimes it is needed to fix some "side" errors like
+rebuilding `libyajl` by hands (it is located in `tmp-bootstrap/libyajl`
+subdirectory.
+
+Last check if binary has been produced can be made:
```
Comment (Bee-lee) [inline docs/local_setup_bare_metal.md:269]: `Last, check if the binary has been produced`? 
 Or maybe `Finally, ...` ?

**Review by Bee-lee — APPROVED** [review]
LGTM overall, just some nitpicks

File: docs/local_setup_bare_metal.md (hunk 1 above)
Comment (Bee-lee) [reply docs/local_setup_bare_metal.md:392]: also maybe this step is not needed at all, because `migrate latest` is an idempotent operation

Comment (tisnik) [reply docs/local_setup_bare_metal.md]: Fixed.

File: docs/local_setup_bare_metal.md [hunk 3]
```diff
@@ -11,4 +11,553 @@ nav_order: 14
...
+
+Try to build Kafkacat/Kcat:
+
+```
+$ ./bootstrap.sh
+```
+
+Please note that sometimes it is needed to fix some "side" errors like
+rebuilding `libyajl` by hands (it is located in `tmp-bootstrap/libyajl`
+subdirectory.
```
[the remark this answers is not in the document]
Comment (tisnik) [reply docs/local_setup_bare_metal.md]: Fixed.

File: docs/local_setup_bare_metal.md [hunk 4]
```diff
@@ -264,9 +264,9 @@ $ ./bootstrap.sh
 
 Please note that sometimes it is needed to fix some "side" errors like
 rebuilding `libyajl` by hands (it is located in `tmp-bootstrap/libyajl`
-subdirectory.
+subdirectory).
 
-Last check if binary has been produced can be made:
+Finally, check if binary has been produced can be made:
```
Comment (Bee-lee) [inline docs/local_setup_bare_metal.md:269]: sorry, I meant that the "can be made" is redundant, right?

Comment (tisnik) [reply docs/local_setup_bare_metal.md:269]: it is, indeed

Comment (tisnik) [reply docs/local_setup_bare_metal.md]: ok second attempt :)

**Pull request merged**